### PR TITLE
Fix horizontal scroll(responsive design)

### PR DESCRIPTION
**Pull request by VitalikVoicix**

### CODE REVIEW

File: src/components/app-header/app-header.styles.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
 import { makeStyles } from '@material-ui/core/styles';
+import { changeCurrency } from '../../redux/currency/currency.actions';
```
Comment (DmytroDidukh) [inline src/components/app-header/app-header.styles.js:2]: For what you need those import here?

File: src/containers/cabinet/cabinet.styles.js [hunk 2]
```diff
@@ -7,6 +7,12 @@ export const useStyles = makeStyles(() => ({
     position: 'relative',
     cursor: 'pointer',
     zIndex: 2,
+
+    '@media (max-width: 500px)': {
+      height: '2rem',
```
Comment (DmytroDidukh) [inline src/containers/cabinet/cabinet.styles.js:12]: Further try to use `rem` and `em` for their intended purpose - sizing fonts

Comment (VitalikVoicix) [reply src/containers/cabinet/cabinet.styles.js:12]: Ок,for the next time I will use  rem or em sizes.

File: src/components/app-header/app-header.styles.js (hunk 1 above)
Comment (VitalikVoicix) [reply src/components/app-header/app-header.styles.js]: Its my mistake,I have allready remove this unusable import

**Review by DmytroDidukh — APPROVED** [review]

**Review by TorskyM — APPROVED** [review]

**Review by maksym-strus — APPROVED** [review]

**Comment by VitalikVoicix** [comment]
I let you know when this branch wiil be ready.I fixed the conflict but instead have another problem.

**Pull request merged**